packages/core/src/config/config.ts

```diff
@@ -963,6 +966,10 @@ export class Config {
     return this.interactive;
   }
 
+  getPtyInfo(): string {
```

Comment:
What do you think about renaming this to `getInteractiveShellEnabled` and making the logic understand the ptyInfo + the interactive property? That way anything trying to understand support for interactive shell pulls from the same context?